src/app.tsx

```diff
@@ -127,47 +128,63 @@ export class AppState extends EventEmitter<AppStateEvents> {
         if (this.#initPromise)
             return this.#initPromise;
 
+        const init_connection = async (connectionName: ConnectionName) => {
+            try {
+                await getLibvirtVersion({ connectionName });
+                const promises = await getApiData({ connectionName });
+                const errorMsgs = promises
+                        .filter(promise => promise.status === 'rejected')
+                        .map(promise => promise.reason.message);
+                if (errorMsgs.length > 0) {
+                    addNotification({
+                        text: _("Failed to fetch some resources"),
+                        detail: errorMsgs.join(', ')
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2465-e2e085cb-20251231-203352-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)